web/.storybook/manager.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * @file Storybook manager configuration.
+ *
+ * @import { ThemeVarsPartial } from "storybook/internal/theming";
+ */
+import { createUIThemeEffect, resolveUITheme } from "@goauthentik/web/common/theme.ts";
```

Comment:
Using `.ts` seems to clue Storybook's bundler into consistently resolving submodules.